### PR TITLE
Add colorMap property to ImageView

### PR DESCRIPTION
**Pull request by mattgibbs**
Adds a pyqtProperty to ImageView that lets you select colormap presets from an enum.  This will close issue #142.

### CODE REVIEW

File: pydm/widgets/image.py [hunk 1]
```diff
@@ -116,17 +120,44 @@ def setColorMapLimits(self, mn, mx):
...
+        """
+        The color map used by the ImageView.
+
+        Parameters
+        -------
+        new_cmap : PyDMColorMap
+        """
+        self.setColorMapToPreset(new_cmap)
+
+    def setColorMapToPreset(self, cmap):
```
Comment (hhslepicka) [inline pydm/widgets/image.py]: Is this method from Qt or PyQt somehow? If not, do you mind changing it to the Pythonic way?

**Comment by hhslepicka** [comment]
Looks great to me @mattgibbs ! Thank you for looking into this one! 
The Q_ENUMS are really an awesome feature! 🥇 
I just left a comment in the review and if it does not make sense feel free to go ahead and merge it.

**Comment by mattgibbs** [comment]
@hhslepicka Good idea.  Done!

**Comment by hhslepicka** [comment]
Perfect @mattgibbs ! Merging in!

**Pull request merged**